### ISSUE TITLE
Suggestion: Improve upload log functionality

### ISSUE THREAD

**Issue by tDegan** [RimSort/RimSort#166]
**Is your feature request related to a problem? Please describe.**
Clicking a button and a link magically appears in your clipboard is not intuitive and error prone.

**Describe the solution you'd like**
When clicking the upload log button a new window opens, which shows the status of the upload.
- On completion the link appears and can then be copied manually. (With a "copy" button maybe, but in any case, show the actual link)
- On Error, show an error message of some kind. At minimum, there should a two distinct error messages: "Log not found" and "Upload failed", so the user knows whether it's a local or a network problem.

**Describe alternatives you've considered**
Just show the path to the log file and let the user manually upload the log to github.

**Additional context**
For me clicking the upload log button freezes the entire application. I know my internet connection is sometimes unreliable, so it may just take very long, but I literally have no indication on what's happening.

**Comment by Limfjorden** [comment]
Isn't this implemented now?

**Comment by hk21702** [comment]
This should be implemented, along with threaded uploading, so RimSort doesn't freeze on a slow/unreliable upload. #446
